doc/releasenotes/10_0_3_release_notes.md

```diff
@@ -0,0 +1,19 @@
+# Release of Vitess v10.0.3
+## Announcement
+
+This patch is providing an update regarding the Apache Log4j security vulnerability (CVE-2021-44228) (#9363), along with a few bug fixes.
+
+ ------------
+## Changelog
+
+### CI/Build
+#### Build/CI
+CI: ubuntu-latest now has MySQL 8.0.26, let us override it with latest 8.0.x #9375
+### Internal Cleanup
+#### Java
+* build(deps): bump log4j-api from 2.13.3 to 2.15.0 in /java #9363
+
+
+The release includes 170 commits (excluding merges)
+
+Thanks to all our contributors: @GuptaManan100, @aquarapid, @askdba, @deepthi, @dweitzman, @dyv, @frouioui, @guidoiaquinti, @harshit-gangal, @jmoldow, @rohit-nayak-ps, @shlomi-noach, @sougou, @systay
```

Comment:
Same comments apply to this PR as my review in https://github.com/vitessio/vitess/pull/9381